api/index.js

```diff
@@ -5,253 +5,224 @@
 /* STATES THE FUNCTIONS WE WILL USE */
 
 const express = require('express') // the way you include libraries, aka code other people made. Express is our webserver thingy.
-const CRUT = require('ssb-crut') // loads module "crut" which is used by Ahau, to make it easy to create and update posts, similar to gatherings in current SSB stack, compared to static messages
-
-const SSB = require('../ssb') // ".." goes backwards to the ssb.js file that is in the main folder
-const podcastModel = require('./podcast-model') // related to the crut framework, it tells the crut how it should be able to work with "podcasts", could also be "gatherings" or "posts", hypothetically. It automatically knows that it's a .json so no need to write out
-
-const commentModel = require('./commentModel') // links to the dependancies for the object Comment (which is created later on) in this case "commentModel"
-const { length } = require('ssb-db2')
 
 // all above is just setting up what we're using, the following is starting to use it
 
-/* ASSIGNS NAMES TO FUNCTIONS */
-
-const app = express() // app is now assigned to express() as a funcion
-
-app.use(express.json()) // express now understands .json
-
-const ssb = SSB() // continues naming our functions
-const podcast = new CRUT(ssb, podcastModel) // new calls upon crut to create a new thing, in this case a podcastModel
-const commentCrut = new CRUT(ssb, commentModel) // 000 why are there no ; after each line? Not needed?
+module.exports = function startAPI (ssb, PORT = 3000) {
```

Comment:
I turned this api into a function `startAPI` so that I could pass in a custom ssb (a temporary testing one).

I also attached `podcast` to `ssb` so you can get it at `ssb.podcast` (same with `comment`). I just stapled them to the side there because it means this file is just deals with API, and the ssb does all the database helper setup